src/legacy/status_im/data_store/chats_test.cljs

```diff
@@ -41,3 +41,13 @@
                        :timestamp                2}]
     (testing "from-rpc"
       (is (= expected-chat (chats/<-rpc chat))))))
+
+(deftest test-decode-chat-id
```

Comment:
[Optional] Just a few suggestions for improved readability:

- Break apart the `chat-id` string to avoid the weird zprint formatting where `let` has a line break before the bindings' vector appears.
- Remove the unnecessary `testing` macro from the test because it's redundant. We know what we are testing here because of the name passed to `deftest`.
- It's more common in Clojure to suffix the test var with `-test` instead of having that as a prefix.

```clojure
(deftest decode-chat-id-test
  (let [community-id "0x0322b9b84acb1631d6a31d41d9cf8e1938d352a624bc130de92869e025c7ca79c4"
        channel-id   "02081bc7-20e4-4362-99e2-37669c28cc08"
        chat-id      (str community-id channel-id)]
    (is (= {:community-id community-id
            :channel-id   channel-id}
           (chats/decode-chat-id chat-id)))))
```